CMakeLists.txt

```diff
@@ -587,27 +569,13 @@ include (${HDF5_VOL_REST_RESOURCES_DIR}/HDFCompilerFlags.cmake)
 #endif ()
 
 #-----------------------------------------------------------------------------
-# Build HDF5, unless the user has specified to use
-# a pre-built HDF5 distribution
+# Include HDF5 Directories
 #-----------------------------------------------------------------------------
-if (NOT PREBUILT_HDF5_DIR AND NOT HDF5_FOUND)
-  include_directories(${HDF5_VOL_REST_SRC_DIR}/${HDF5_DIR_NAME}/src)
-  include_directories(${HDF5_VOL_REST_SRC_DIR}/${HDF5_DIR_NAME}/src/H5FDsubfiling)
-  add_subdirectory(${HDF5_DIR} ${PROJECT_BINARY_DIR}/${HDF5_DIR_NAME})
-elseif (NOT PREBUILT_HDF5_DIR AND HDF5_FOUND)
-  set (HDF5_LIBRARIES_TO_EXPORT
-    ${HDF5_C_HL_LIBRARIES}
-    ${HDF5_C_LIBRARIES}
-  )
-else ()
-  link_directories(${PREBUILT_HDF5_DIR}/lib)
-  set (HDF5_LIBRARIES_TO_EXPORT
-       libhdf5.so
-       libhdf5_hl.so
-       libhdf5.a
-       libhdf5_hl.a
-  )
-endif ()
+include_directories(${HDF5_INCLUDE_DIRS})
+
+list (APPEND HDF5_LIBRARIES_TO_EXPORT
+  ${HDF5_C_HL_LIBRARIES}
```

Comment:
Looks like `HDF5_C_LIBRARIES` probably needs to be added back here, but also looks like the list being appended to should be `HDF5_VOL_REST_LIBRARIES_TO_EXPORT` instead of `HDF5_LIBRARIES_TO_EXPORT` since the latter would be something HDF5 would set